lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -484,12 +483,12 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
           Value shuffledPaddingList =
               createConstantIntList(binder, rewriter, padding);
           Value zero;
-          if (resultType.getDtype().isa<FloatType>()) {
+          if (resultTypeOut.getDtype().isa<FloatType>()) {
             zero = rewriter.create<Torch::ConstantFloatOp>(
                 binder.getLoc(), rewriter.getType<Torch::FloatType>(),
```

Comment:
Why does the ConstantFloatOp builder require Torch::FloatType passed, but ConstantIntOp doesn't require Torch::IntType?

Reply:
Actually, I haven't added that code, it was added by @rsuderman. As per my understanding, `Torch::FloatType` is also not required.